src/services/parser/partytimeParser.ts

```diff
@@ -0,0 +1,179 @@
+    generator: feed.generator,
+    guid: feed.guid,
+    imageURL: feed.itunesImage || feed.image?.url,
+    itunesType: feed.itunesType || podcastItunesTypeDefaultValue,
+    language: feed.language,
+    lastBuildDate: feed.lastBuildDate,
+    link: feed.link,
+    liveItems: feed.podcastLiveItems ?? [],
+    medium: feed.medium ?? Phase4Medium.Podcast,
+    owner: feed.owner,
+    pubDate: feed.pubDate,
+    subtitle: feed.subtitle,
+    summary: feed.summary,
+    title: feed.title,
+    type: feed.itunesType,
+    value: feed.value ? [valueCompat(feed.value)] : []
+  }
+}
+
+// Convert the podcast-partytime schema to a podverse compatible schema.
+const episodeCompat = (episode: Episode) => {
+  return {
+    alternateEnclosures: episode.alternativeEnclosures ?? [],
+    author: [episode.author],
+    chapters: episode.podcastChapters,
+    // TODO: why does contentLinks exist on liveItem but not episode type?
+    contentLinks: episode.contentLinks || [],
+    description: episode.content || episode.description,
+    duration: episode.duration,
+    enclosure: episode.enclosure,
+    explicit: episode.explicit,
+    // funding: Array.isArray(episode.podcastFunding) ? episode.podcastFunding?.map((f) => fundingCompat(f)) : [],
+    guid: episode.guid,
+    imageURL: episode.image,
+    itunesEpisode: episode.podcastEpisode?.number || episode.itunesEpisode,
+    itunesEpisodeType: episode.itunesEpisodeType,
+    itunesSeason: episode.podcastSeason?.number || episode.itunesSeason,
+    link: episode.link,
+    pubDate: episode.pubDate,
+    socialInteraction: episode.podcastSocialInteraction ?? [],
+    soundbite: episode.podcastSoundbites ?? [],
+    subtitle: episode.subtitle,
+    summary: getLongerSummary(episode.content, episode.description),
+    title: episode.title,
+    transcript: episode.podcastTranscripts ?? [],
+    value: episode.value ? [valueCompat(episode.value)] : []
+  } as ParsedEpisode
+}
+
+// TODO: there is duplication here with episodeCompat...can we reuse?
+const liveItemCompatToEpisode = (liveItem: ExtendedPhase4PodcastLiveItem) => {
+  const getChatEmbedUrlValue = (chat?: ExtendedChat) => {
+    if (chat?.phase === 'pending' && chat.embedUrl) {
+      return chat.embedUrl
+    }
+    // deprecated embed value
+    else if (chat?.phase === '4' && chat.url) {
+      return chat.url
+    }
+    return ''
+  }
+
+  return {
+    chat: getChatEmbedUrlValue(liveItem.chat),
+    end: liveItem.end,
+    episode: episodeCompat({
+      ...liveItem,
+      contentLinks: liveItem.contentLinks, // TODO:
+      duration: 0,
+      explicit: false, // TODO: liveItem.explicit
+      socialInteraction: [],
+      soundbite: [],
+      subtitle: '', // TODO: liveItem.subtitle
+      summary: '', // TODO: liveItem.summary,
+      transcript: []
+    }),
+    start: liveItem.start,
+    status: liveItem.status?.toLowerCase()
+  }
+}
+
```

Comment:
again, I'd create a `./compat.ts` file or something. This will force you to export them and then if you ever want to, they're easier to test.